worker.py

```diff
@@ -95,11 +95,13 @@ def run(self):
         service = config.ACCOUNTS[self.worker_no][2]
         while True:
             try:
-                self.api.login(
+                loginsuccess = self.api.login(
                     provider=service,
                     username=config.ACCOUNTS[self.worker_no][0],
                     password=config.ACCOUNTS[self.worker_no][1],
                 )
+                if not loginsuccess:
+                    raise pgoapi_exceptions.AuthException
```

Comment:
You raise an exception here, which will be caught literally in the next line. Better to just set error code and restart in this if:

```py
if not loginsuccess:
    self.error_code = 'LOGIN FAIL'
    self.restart()
    return
```